### PR TITLE
Configure vscode to run pytest

### PR DESCRIPTION
**Pull request by ross-p-smith**
This PR relates to #69 and allows the developers to be able to run the tests from the vscode test runner

## Does this introduce a breaking change?
```
[ ] Yes
[x] No
```

## Pull Request Type
```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[x] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What to Check
- Open the code from vscode
- Can you run the tests from the "Launch" menu

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -56,6 +48,20 @@
             "skipFiles": [
                 "node_modules/**"
             ],
+        },
+        {
+            "name": "Debug tests",
```
Comment (adamdougal) [inline .vscode/launch.json:53]: This didn't work for me. I got the error:
```
/usr/local/bin/python: No module named pytest
```

I couldn't see it in the requirements at all, do we need to add it there?

Comment (ross-p-smith) [reply .vscode/launch.json:53]: Thanks, I'll wipe everything and check again as it may have been an extension that installed it!

Comment (ross-p-smith) [reply .vscode/launch.json:53]: Looks like the python extension installs

**Review by adamdougal — APPROVED** [review]
lgtm

**Pull request merged**